### PR TITLE
Upgrade vulkan SDK to 1.3.224

### PR DESCRIPTION
**Pull request by gadzooka**
- Upgrade SDK to 1.3.224.
- Isolate vulkan prebuilt headers so they don't interfere with local SDK build when present in hunter cache.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -24,22 +24,19 @@ if (CMAKE_CROSSCOMPILING)
     # Per non-development platform configuration
     if (IOS OR TVOS)
         # The iOS/tvOS libraries must be lipo'ed together when using legacy build systems.
+        # Note: The vulkan sdk ios-arm64_x86_64-simulator/libMoltenVK.a already contains an arm64 architexture which must be replaced with ios-arm64/libMoltenVK.a
```
Comment (mathieumallet) [inline CMakeLists.txt]: *architecture

File: other/glad/include/vulkan/vulkan.h [hunk 2]
```diff
@@ -0,0 +1,4318 @@
+/**
```
Comment (mathieumallet) [inline other/glad/include/vulkan/vulkan.h:1]: glad/vulkan.h and vulkan/vulkan.h.. confusing

**Review by mathieumallet — APPROVED** [review]
+1

File: other/glad/include/vulkan/vulkan.h (hunk 2 above)
Comment (gadzooka) [reply other/glad/include/vulkan/vulkan.h:1]: They're identical... just needed to satisfy how different cpp files include this header, and this was the simplest way.

Comment (mathieumallet) [reply other/glad/include/vulkan/vulkan.h:1]: maybe it'd be better to copy it as part of the cmake hunter build (so there's only one copy in the repo)

**Pull request merged**